e2e/run_test.go

```diff
@@ -263,7 +263,7 @@ func TestRunSpecialCharactersInPrefix(t *testing.T) {
 	defer file.Remove()
 
 	cmd := s5cmd("run", file.Path())
-	cmd.Timeout = time.Second
+	cmd.Timeout = time.Second * 3
```

Comment:
Could you revert this change please? We'd like to fix test timeout issues in another PR where we set the `cmd.Timeout` in the `s5cmd` helper function.

Reply:
sure, I will revert it and let me know once your PR is merged, I will rebase it.